script/generate_async.py

```diff
@@ -99,7 +143,29 @@
 
         top_level_content_patch.append(line)
 
-    with open(f"{BASE_PATH}/grafana_client/elements/_async/__init__.py", "w") as fp:
+    with open(f"{target}/__init__.py", "w") as fp:
         fp.write("\n".join(top_level_content_patch) + "\n")
 
-    subprocess.call(["poe", "format"])
+    subprocess.call(["black", target])
+    subprocess.call(["isort", target])
+
+    exitcode = 0
+
+    if run_check:
+        command = f"diff -u {TARGET} {target}"
```

Comment:
I tested this locally, and found that it yield a diff constantly, investigating.

Reply:
I tried a poor-mans version, and it worked for me, running macOS. Maybe we need to do it differently, resorting to pure Python?

Reply:
Operating system / newline representation / filesystem permission woes? Stray files in tree?

Reply:
I am on Linux Fedora.
The issue was that python byte code cache is included by accident in the diff. Proposed a fix for it.